src/components/masters/masters-page/MastersPage.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { AppContext } from '../../common/app-context';
+
+const MastersPage = ({
+  children,
+  pageContext,
+  username,
+}) => (
+  <AppContext.Provider
+    value={{
+      header: {
+        userMenu: [
```

Comment:
Passing all the header menu items in this sort of config gives more control to masters & enterprise to control their own individual menu options, including being able to order the links as needed.

Reply:
👍